tests/dialects/test_snowflake.py

```diff
@@ -52,6 +52,7 @@ def test_snowflake(self):
         self.validate_identity("SELECT BOOLAND(1, -2)")
         self.validate_identity("SELECT BOOLXOR(2, 0)")
         self.validate_identity("SELECT BOOLOR(1, 0)")
+        self.validate_identity("SELECT IFF(condition, value1, value2)")
```

Comment:
@fivetran-felixhuang let's please check for existing tests to avoid introducing duplicates. The `IFF` function is already tested for roundtrip:

```python
        self.validate_all(
            "SELECT IFF(TRUE, 'true', 'false')",
            write={
                "snowflake": "SELECT IFF(TRUE, 'true', 'false')",
                "spark": "SELECT IF(TRUE, 'true', 'false')",
            },
        )
```

```suggestion
```

Reply:
Since the snippet I shared lives within `TestSnowflake`, the first argument is always parsed according to the Snowflake dialect, so having a `"snowflake": ...` entry in `write` means we also test the roundtrip.